### DISCUSSION TITLE
Help needed to understand Chia

### DISCUSSION THREAD

**Discussion by lexick2016** [Chia-Network/chia-blockchain#2375] [Q&A]
Can someone explain to me aft the plots are going into the temp ssd/space then it moves on the Permanent hdd (in my case my nas).
How long it stays there?
How much space you need on the permanent hdds?
What happens when the hdds(permanent ones) are full? You stop making money? You are still making money? Are you still plotting on the ssd?
I would really appreciate your help to understand this. Thank you

**Comment by GrayXu** [comment]
check wiki [https://github.com/Chia-Network/chia-blockchain/wiki/k-sizes](https://github.com/Chia-Network/chia-blockchain/wiki/k-sizes)

**Reply by lexick2016** [reply]
GrayXu thank you for taking the time to help me.
I had but it won’t give me the answer I am searching for.

**Reply by al-dev112233** [reply]
The plot files stay there until you delete them. You need as much space as you want. Just stop adding plot files.
When HDD is full: you cannot add new plot files to them. You can still use the existing plots.
SSD: is only used in your setup during the creation of new plot files.

**Reply by lexick2016** [reply]
Thank you for taking the time.
Let me tell you what I understood..
The CPU/Ram creates the plots on the ssd and then puts them on the hdd(permanent).
My question is: are we paid only on the new plots that we create, or are we paid (in crypto of course) on what is permanently stored on the hdd?
So basically if I have a 20Tb hdd once full, as the machine will not create new polts, will it still produce money or it’s better to delete the permanent pots to mine for new ones.
Hope you understand what I am trying to say...

**Reply by fabitom** [reply]
Farming is based on generated plots, that is all. You need share your plots.

**Reply by al-dev112233** [reply]
You have to let the generated plot files "farm the chia coins". Do not touch these 100 GB files. You will have to generate and add more plot files over time because the whole network gets bigger in terms of petabyes so your fraction (a few terabytes) is getting lower in percentage overall. You will make less and less coins over time **if your farm does not grow**.

**Reply by lexick2016** [reply]
So basically I can have one strong machine to create plots and them practically just add hdd for the farm, to
add more and more space.
My question is can you move those created plots to another machine? Another PC that is lower on power just to leave the farms work they way, or they need to be kept on the original machine?

**Reply by al-dev112233** [reply]
You can add a directory that has finished plots in it. I assume that you can import plots that way that were initially created on other machines.
Another way would be to write the plot files directly to the disks of the farming machine via network share. (= final directory)

**Reply by lexick2016** [reply]
That seems amazing.
Thank you and the rest for taking the time to help.
I guess we will learn as we go in the end as this is kind of a new thing for most of us.

**Reply by lexick2016** [reply]
Fair I understand that English is not your first language and you user Google translate but I can’t understand what you are trying to say. Maybe someone else will. I wish I could help you
